public/scripts/components/sub-components/PicTitleThumb.js

```diff
@@ -5,10 +5,10 @@ export default React.createClass({
 	render: function() {
 		return (
 		<div className="story-thumb-box" >
-			<Link to="/details">
+			<Link to="/stories/:storyId/details">
```

Comment:
in this case you will be linking to the LITERAL string `'/stories/:storyId/details'`.  You probably want to link to something like `'/stories/3/details'`, for example.